### PR TITLE
106512/legal requirements change buttons

### PR DESCRIPTION
**Pull request by dneed-nimble**

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalConsultation.cshtml.cs [hunk 1]
```diff
@@ -31,9 +31,15 @@ public async Task<IActionResult> OnPostAsync(int id)
 				nameof(ThreeOptions.NotApplicable) => ThreeOptions.NotApplicable,
 				_ => LegalRequirements.ConsultationDone
 			};
-
 			await LegalRequirementsRepository.UpdateByProjectId(id, LegalRequirements);
 
+			var (returnPage, fragment, back) = GetReturnPageAndFragment();
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalConsultation.cshtml.cs:36]: Could move this into the LegalModelBase.cs

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalFoundationConsent.cshtml.cs [hunk 2]
```diff
@@ -32,6 +32,12 @@ public async Task<IActionResult> OnPostAsync(int id)
 				_ => LegalRequirements.FoundationConsent
 			};
 
+			var (returnPage, fragment, back) = GetReturnPageAndFragment();
+			if (ReturnPage(returnPage))
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalFoundationConsent.cshtml.cs:36]: Could move this into the LegalModelBase.cs

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalGoverningBody.cshtml.cs [hunk 3]
```diff
@@ -34,6 +34,12 @@ public async Task<IActionResult> OnPostAsync(int id)
 
 			await LegalRequirementsRepository.UpdateByProjectId(id, LegalRequirements);
 
+			var (returnPage, fragment, back) = GetReturnPageAndFragment();
+			if (ReturnPage(returnPage))
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalGoverningBody.cshtml.cs:38]: Could move this into the LegalModelBase.cs

**Review by jordan-beard — APPROVED** [review]

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalConsultation.cshtml.cs (hunk 1 above)
Comment (dneed-nimble) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalConsultation.cshtml.cs]: True!

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalFoundationConsent.cshtml.cs (hunk 2 above)
Comment (dneed-nimble) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalFoundationConsent.cshtml.cs]: True!

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalGoverningBody.cshtml.cs (hunk 3 above)
Comment (dneed-nimble) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalGoverningBody.cshtml.cs]: True!

**Pull request merged**